project/R/conceptSets.R

```diff
@@ -78,6 +78,113 @@ conceptSets <- list(
+    ),
+    
+    UC3LabMeasurements = c(
+      3024942, # source: factor VIII inhibitor titer /// concept: coagulation factor VIII inhibitor [Units/volume] in platelet poor plasma by coagulation assay
+      3022520, # source: factor VIII activity measurement /// concept: coagulation factor VIII activated [Units/volume] in platelet poor plasma by coagulation assay
+      3011832, # source: factor VIII activity measurement /// concept: coagulation factor VIII activity [Units/volume] in platelet poor plasma by Chromogenic assay
+      43534000, # source: Von Willebrand factor activity measurement /// concept: von Willebrand factor (vWf) activity [Units/volume] in platelet poor plasma by Immunoassay
+      3002124, # source: Von Willebrand factor antigen measurement /// concept: von Willebrand factor (vWf) Ag [Units/volume] in platelet poor plasma by Immunoassay
+      3023693, # source: Von Willebrand factor propeptide measurement /// concept: von Willebrand factor (vWf) multimers in platelet poor plasma by Immunoblot
+      3042349, # source: Von Willebrand factor propeptide measurement /// concept: von Willebrand factor (vWf) cleaving protease inhibitor [Units/volume] in platelet poor plasma
+      4175016, # source: activated partial thromboplastin time (APTT) /// concept: partial thromboplastin time, activated
+      3034426, # source: Prothrombin time (PT) /// concept: prothrombin time (PT)
+      37393863, # source: platelet count /// concept: platelet count
+      3009542, # source: hematocrit /// concept: Hematocrit [Volume Fraction] of Blood
+      3016407, # source: fibrinogen /// concept: fibrinogen [mass/volume] in platelet poor plasma by coagulation assay
+      3006923, # source: ALT /// concept: alanine aminotransferase [enzymatic activity/volume] in serum or plasma
+      3013721 # source: AST /// concept: aspartate aminotransferase [enzymatic activity/volume] in serum or plasma
+    ),
+    
+    UC3Treatments = c(
+      35766745, # source: factor VIII /// concept: factor VIII 250 UNT Injection [Advate]
+      21154209, # source: factor VIII /// concept: factor VIII 250 UNT Injection [Kogenate Bayer]
+      35831239, # source: factor VIII /// concept: factor VIII 250 UNT Injection [Eloctate]
+      40745282, # source: factor VIII /// concept: emicizumab 150 MG/ML Injectable Solution [Hemlibra]
+      40492862, # source: factor VIII dose is continuous infusion or bolus dose /// concept: mode of drug administration
+      4129275, # source: factor VIII dose is continuous infusion or bolus dose /// concept: continuous infusion
+      4265597, # source: factor VIII dose is continuous infusion or bolus dose /// concept: by bolus infusions
+      1517070 # source: desmopressin /// concept: desmopressin
+      # TODO FIGURE OUT IF DESCENDENTS WORK WITH THESE CONCEPTS 
+      # TODO MAP REMAINING DRUG BRANDS 
+    ),
+    
+    UC3Surgery = c(
+      45888085, # source: Medical/ surgical procedure /// concept: medical procedure
```

Comment:
This concept is a CPT4 classification code for [Surgery](https://athena.ohdsi.org/search-terms/terms/45888085). Use instead [Therapeutic procedure](https://athena.ohdsi.org/search-terms/terms/4172515)